lib/connectors/kv-memory.js

```diff
@@ -0,0 +1,139 @@
+'use strict';
```

Comment:
I wonder why we don't use https://github.com/isaacs/node-lru-cache as the base for implementing in-memory KV store.